cmd/root.go

```diff
@@ -15,16 +15,16 @@ var rootCmd = &cobra.Command{
 
 		_, err := docker.NewEnvClient()
 		if err != nil {
-			panic(err)
+			log.Fatal(err)
 		}
 
-		fmt.Println("Dunner running!")
+		log.Println("Dunner running!")
```

Comment:
This makes the output to be `INFO[0000] Dunner running!` which is not nice, Shall we keep these as it is?

Reply:
Alright.